src/components/migrationToV3.stories.mdx

```diff
@@ -609,11 +748,11 @@ const App = () => (
 );
 
 const MyChild = () => {
-  const selectVideoInput = useSelectVideoInputDevice();
+  const meetingManager = useMeetingManager();
 
   const handleClick = async (): => {
     try {
-      await selectVideoInput('deviceId');
+      await meetingManager.startAudioInputDevice('deviceId');
     } catch (error) {
       // Handle device selection error.
       console.error('Failed to select video input device', error);
```

Comment:
Have to update the error message since you use `meetingManager.startAudioInputDevice`.

Reply:
Updated to `meetingManager.startVideoInputDevice`